src/php/lib/Grpc/ServerCallWriter.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Grpc;
+
+/**
+ * This is an experimental and incomplete implementation of gRPC server
+ * for PHP. APIs are _definitely_ going to be changed.
+ *
+ * DO NOT USE in production.
+ */
+
+class ServerCallWriter
+{
+    public function __construct($call)
+    {
+        $this->call_ = $call;
+    }
+
+    public function start(
+        array $initialMetadata,
+        \Google\Protobuf\Internal\Message $data = null,
```

Comment:
As discussed, can we try to not type-hint something with the explicit class of `Google\Protobuf\Internal\Message`? And just assume `$data` is a protobuf object for now? Type-hinting something with `Internal\` in it _feels_ wrong.

Reply:
type-hint removed